client/transaction-pool/src/api.rs

```diff
@@ -67,13 +69,9 @@ impl<Client, Block> FullChainApi<Client, Block> {
 
 		FullChainApi {
 			client,
-			pool: ThreadPoolBuilder::new()
-				.pool_size(2)
-				.name_prefix("txpool-verifier")
-				.create()
-				.expect("Failed to spawn verifier threads, that are critical for node operation."),
```

Comment:
Before we spawned `2` threads and now you spawn unlimited of them...

Reply:
It is still going to be limited by the number of blocking threads, no?
If it's not limited then we might be generating quite huge load on the entire machine due to revalidation (which is not bounded currently) and that might interfere with other tasks.

Reply:
The limit is quite high, I don't remember it, but it is way more than 2